components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -5325,27 +5325,20 @@ public void updatePermissionListOfGroup(String groupName, String[] permissionToA
         return filteredFlatAttributeMap;
     }
 
-    private Map<String, Attribute> getFilteredEnterpriseUserSchemaAttributes(Map<ExternalClaim, LocalClaim>
+    /**
+     * Get filtered claims based on property `isSupportedByDefault` of mapped local dialect claim.
+     *
+     * @param scimClaimToLocalClaimMap
+     * @return
+     */
+    private Map<String, Attribute> getFilteredSchemaAttributes(Map<ExternalClaim, LocalClaim>
                                                                                      scimClaimToLocalClaimMap) {
-
```

Comment:
Have a newline after method signature

Reply:
Fixed in commit 950abcf003d33bdb50b455a0aa3df16e33f27a5b